test/config/utility.cc

```diff
@@ -347,11 +351,11 @@ std::string ConfigHelper::adsBootstrap(const std::string& api_type) {
       address: 127.0.0.1
       port_value: 0
 )EOF",
-      TestEnvironment::nullDevicePath(), api_type);
+      api_type, TestEnvironment::nullDevicePath());
 }
 
-envoy::config::cluster::v3::Cluster ConfigHelper::buildCluster(const std::string& name, int port,
-                                                               const std::string& ip_version) {
+envoy::config::cluster::v3::Cluster
+ConfigHelper::buildStaticCluster(const std::string& name, int port, const std::string& address) {
```

Comment:
What do you think about combining these two cluster builders with a boolean for static/dynamic? In the spirit of
https://github.com/envoyproxy/envoy/issues/6327

Reply:
It might become less readable as they take different arguments?  I could just overload buildCluster either, what do you think?

Reply:
Hmmmm. I see. Can you add the TODO here and maybe just stick with combining the cluster/rediscluster

> I could just overload buildCluster either

What do you mean?

Reply:
Just rename `buildStaticCluster` to `buildCluster` so that it returns a static or dynamic cluster depending on what arguments it gets

Reply:
Yeah I think that would be a little confusing if they were both named buildCluster. Unused/optional args would probably be clearer